hathor/manager.py

```diff
@@ -1094,6 +1127,21 @@ def is_healthy(self) -> Tuple[bool, Optional[str]]:
 
         return True, None
 
+    def check_sync_state(self):
+        now = time.time()
+
+        if self.has_recent_activity():
+            self.first_time_fully_synced = now
+
+            total_sync_time = LogDuration(self.first_time_fully_synced - self.start_time)
+            vertex_count = self.tx_storage.get_tx_count() + self.tx_storage.get_block_count()
+
+            # Changing the fields in this log could impact log collector that parse them
+            self.log.info('reached synced state for the first time', total_sync_time=total_sync_time,
```

Comment:
Maybe say "has recent activity for the first time"? I feel the log might mislead us to think the full node is synced while the criteria is related to recent activity. Note that each sync algorithm might have a different criteria for what being synced means.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/490/commits/0628898e0e618ecc70ea8f92a3ab5fe9f49d58f9

Reply:
Thanks!